### PR TITLE
Feat : Added Auth Pages

### PR DESCRIPTION
**Pull request by shlok30**
Hi Team,

Can you please review my auth pages

login : https://compassionate-mclean-2c6cf6.netlify.app/login.html
Sign Up : https://compassionate-mclean-2c6cf6.netlify.app/signup.html

The site is not responsive as of now, will work on that later.
Have used a little inline styling in some places to overwrite some utitlity css.

Thanks

### CODE REVIEW

File: signup.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+</head>
+<body>
+
+    <nav class = "navigation flex gap-xl align-center full-width">
+        <div class="nav-brand">Water-Commerce</div>
+        <div class = "input-wrapper flex grow-1 flex-center" >
+            <input class = "input-field" placeholder="Please Enter Text">
+        </div>
+        <div class = "flex gap-xl flex-center grow-1" >
+            <button class="btn btn-secondary">Login</button>
```
Comment (abhishek17551) [inline signup.html:23]: reference this button to login page

File: signup.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            <div class = "primary-text center-text text-m bold">Sign Up</div>
+            <div class = "padding-s">
+                <label for="demo" class="input-label">Email Address</label>
+                <input type = "email" id = "demo" class = "input-field m2-top full-width" style = "display: block;width:100%" placeholder="Please Enter Email Here">
+            </div>
+            <div class="padding-s flex flex-column">
+                <label for="demo" class="input-label">Enter Password</label>
+                <input type = "password" id = "demo" class = "input-field m2-top full-width" style = "display: block;width:100%" placeholder="Please Enter Password Here">
+            </div>
+
```
Comment (abhishek17551) [inline signup.html:41]: would suggest you to add another 'input' for confirm password here.

File: signup.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            <div class="padding-s flex flex-column">
+                <label for="demo" class="input-label">Enter Password</label>
+                <input type = "password" id = "demo" class = "input-field m2-top full-width" style = "display: block;width:100%" placeholder="Please Enter Password Here">
+            </div>
+            
+    
+            <div class = "padding-s-block m2-top flex space-between">
+                <div class="flex gap-m align-center">
+                    <input type = "checkbox">
+                    <p>Remember Me</p>
```
Comment (abhishek17551) [inline signup.html:46]: might not be necessary in signup page.

**Review by abhishek17551 — APPROVED** [review]
few changes to be. rest looks fine.

**Review by pariyar07 — APPROVED** [review]
Looks good to merge

**Pull request merged**